docs/UserGuide.md

```diff
@@ -150,6 +151,17 @@ The parameter `PATIENT_NAME` and `DOCTOR_NAME` are case-sensitive, and must be a
 **Examples:**
 * `add-appt pn/John Doe dn/Jane Doe d/23-04-1987 t/1100`
 
+### Deleting an appointment: `delete-appt`
+
+Deletes the specified appointment from the address book.
+
+Format: `delete-appt UNIQUE_ID`
+
+* Deletes the appointment tagged to the `UNIQUE_ID`.
```

Comment:
Not really sure if we need this line.

Reply:
Oh I followed the additional comments for the `delete` command in the UG, but I agree it's not exactly necessary